webapp/src/pages/Settings.tsx

```diff
@@ -422,15 +440,17 @@ const Settings: React.FC = () => {
               </FormGroup>
             </FormControl>
             <FormControl>
-              {displayPostprocessorToggleSection(pipelineIndex, pipeline)}
+              {pipeline.postprocessors &&
+                displayPostprocessorToggleSection(pipelineIndex, pipeline)}
               <FormGroup sx={{ gap: 2 }}>
                 {pipeline.postprocessors?.map((postprocessor, index) => (
                   <Paper key={index} variant="outlined" sx={{ padding: 2 }}>
                     <Columns columns={3}>
-                      {displayReadonlyFields(
-                        "class_name",
-                        postprocessor.class_name
-                      )}
+                      {postprocessor.class_name &&
+                        displayReadonlyFields(
+                          "class_name",
+                          postprocessor.class_name
+                        )}
```

Comment:
The `class_name` parameter is NOT optional, it must exist.
```suggestion
                      {displayReadonlyFields(
                        "class_name",
                        postprocessor.class_name
                      )}
```

Reply:
For "TemperatureScaling" and "ThresholdConfig," class_name is declared as optional, whereas in "CustomObject," mentioned "class_name" is required. As we do intersection for all three types, will the class_name be considered "required"?

Reply:
I see the generated types are a little bit broken here, saying that `postprocessor.class_name` is `class_name?: undefined`. That's confusing. 😝 I'll see if I can do anything about that.
Basically, when defining the config, one can define a `TemperatureScaling` or a `ThresholdConfig` for which the `class_name` has a default value. When getting the config, it will always have the `class_name` defined.

Reply:
I played with different possibilities and I won't be able to fix the generated type of `postprocessor.class_name`. If it becomes a problem, let me know and we can instead overwrite the generated type manually.

Reply:
Turns out I found a solution! You can search for `postprocessors?:` in 48513d284700d320f5ed19a11de1e952e9dbe224 to see the result.